wiki/Gameplay/Game_modifier/Target_Practice/en.md

```diff
@@ -26,11 +26,17 @@ tags:
 
 *Notice: Enabling the Target Practice mod will result in an unranked play.*
 
-The **Target Practice** mod is an experimental [game modifier](/wiki/Gameplay/Game_modifier) that removes all the mapped [hit objects](/wiki/Gameplay/Hit_object) and replaces them with a simplified and consistent set of "targets".
+The **Target Practice** mod is an experimental [game modifier](/wiki/Gameplay/Game_modifier) for [osu!](/wiki/Game_mode/osu!), which removes all the mapped [hit objects](/wiki/Gameplay/Hit_object) and replaces them with a simplified and consistent set of "targets". Due to this mod's nature, The mod is used mostly for fun, but can also be helpful to players that want to improve themselves, for they can use it to practice maintaining a consistent tempo and improving hit accuracy.
```

Comment:
Consistent might not be the best word because I'm pretty sure they are random?

Reply:
In some ways it may be I think, e.g. the targets don't have so-called *patterns*, and they are placed per full beat.

Random can be used to describe their locations.

Reply:
removing it to avoid misunderstanding, as it is already described below